README.md

```diff
@@ -67,3 +67,17 @@ Note: Try to split the frontend in [components and screens](https://gist.github.
 ### Yet An Otter System
 
 [Site Link](https://proyecto-nutria.github.io/)
+
+### Known issues
```

Comment:
Imo this issues need to be removed from the `package.json`, **Reason** is not used anymore if I'm not wrong (can you confirm @GoberInfinity) and the **Browser** was a missupdate from @searleser97 because is the one that he uses. 😂